sketcherMinimizer.cpp

```diff
@@ -3546,21 +3545,17 @@ void sketcherMinimizer::loadTemplates()
     if (loaded || !m_templates.getTemplates().empty()) {
         return;
     }
-#ifdef BUNDLE_TEMPLATES
     m_templates.getTemplates() = coordgen_templates();
-    for (auto* m: m_templates.getTemplates()) {
+    for (auto* m : m_templates.getTemplates()) {
         normalizeTemplate(m);
     }
-#else
-    string filename = getTempFileProjDir() + "templates.mae";
-
-    loadTemplate(filename, m_templates.getTemplates());
+#ifdef USE_MAEPARSER
 
     filename = getUserTemplateFileName();
     loadTemplate(filename, m_templates.getTemplates());
```

Comment:
Some silly thought for maybe another PR: if we move this a few lines up, before `m_templates.getTemplates() = coordgen_templates();`, do we get to overriding the default templates for free?

Reply:
I didn't want to change user-visible behavior in this PR.

I agree, though, it might make sense for user templates to hit before the default templates.